src/mixins/common-mixin.ts

```diff
@@ -92,12 +92,16 @@ export default class CommonMixin extends Vue {
 
   /** The appropriate edit label for corrections or alterations. */
   get editLabel (): string {
-    return this.isCorrectionFiling ? 'Correct' : 'Change'
+    if (this.isCorrectionFiling) return 'Correct'
+    if (this.isAlterationFiling) return 'Change'
+    return 'Edit'
```

Comment:
I removed the assumption that "if it's not a correction then it's an alteration" and added a filing-agnostic fallback.
Same on line 104 below.